README.md

```diff
@@ -1,2 +1,228 @@
-# satellite6-upgrade
-Satellite 6 upgrade related tools
+# Satellite6-upgrade
+
+Satellite6 upgrade is a set of tools to be used for following reasons:
+1. Creating and deleting satellite/capsule instances using RHEVM templates/Openstack images
+2. Upgrade Satellite6, Capsule and clients (RHEVM instances / Users own setup)
+3. Post Upgrade Satellite Entities Verification
```

Comment:
`4. Performs data validation to ensure the entities created before upgrade exists post upgrade as well`